config/environments/test.rb

```diff
@@ -19,14 +19,15 @@
   # loading is working properly before deploying your code.
   config.eager_load = ENV['CI'].present?
 
-  config.assets_digest = false
+  # Configure public file server for tests with Cache-Control for performance.
+  config.public_file_server.headers = { 'Cache-Control' => "public, max-age=#{1.hour.to_i}" }
```

Comment:
We are using our own middleware for serving static files, should we keep this here as its not used (from what I understand)?